src/stdlib/process.md

```diff
@@ -0,0 +1,60 @@
+---
```

Comment:
:exploding_head: Dude, I am stoked we are adding Node-like stuff for wasi in the stdlib! :smile: :tada: